daffodil-test-ibm1/src/test/resources/test-suite/ibm-contributed/dpacalbin100_01.dfdl.xsd

```diff
@@ -43,7 +44,7 @@
 	<xs:element name="testBinarySeconds" type="xs:dateTime">
 		<xs:annotation>
 			<xs:appinfo source="http://www.ogf.org/dfdl/">
-				<dfdl:element  binaryCalendarRep="binarySeconds" binaryCalendarEpoch="1970-01-01T00:00:00+00:00"/>
```

Comment:
Does including the timezone not work? I think we want to support both with and without the timezone. And since this test came from IBM, we probably want to keep it as similar to the original as possible.